client/my-sites/upgrade-nudge/docs/example.jsx

```diff
@@ -8,30 +8,29 @@ import React from 'react';
  */
 import UpgradeNudge from 'my-sites/upgrade-nudge';
 
-export default React.createClass( {
-
-	displayName: 'UpgradeNudge',
-
-	render: function() {
-		return (
+const UpgradeNudgeExample = () => {
+	return (
+		<div>
+			<div>
+				<UpgradeNudge
+					feature="custom-domain"
+					href="#"
+					shouldDisplay={ () => true }
+				/>
+			</div>
 			<div>
-				<div>
-					<UpgradeNudge
-						feature="custom-domain"
-						href="#"
-						shouldDisplay={ () => true }
-					/>
-				</div>
-				<div>
-					<UpgradeNudge
-						title="This is a title"
-						message="This is a custom message"
-						icon="customize"
-						shouldDisplay={ () => true }
-						compact
-					/>
-				</div>
+				<UpgradeNudge
+					title="This is a title"
+					message="This is a custom message"
+					icon="customize"
+					shouldDisplay={ () => true }
+					compact
+				/>
 			</div>
-		);
-	}
-} );
+		</div>
+	);
+};
+
+UpgradeNudgeExample.displayName = 'UpgradeNudge';
```

Comment:
I think the original is wrong - why name it `UpgradeNudge`? It will look weird in React Tools (`UpgradeNudge` inside an `UpgradeNudge`). I'd leave it as `UpgradeNudgeExample` (the original used `export default React.createClass` so it had to define the `displayName` explicitly, but with your changes, that's no longer needed).